lighthouse-core/config/default.js

```diff
@@ -675,7 +675,6 @@ module.exports = {
         {"id": "first-meaningful-paint", "weight": 5, "group": "perf-metric"},
         {"id": "speed-index-metric", "weight": 1, "group": "perf-metric"},
         {"id": "estimated-input-latency", "weight": 1, "group": "perf-metric"},
-        {"id": "time-to-interactive", "weight": 5, "group": "perf-metric"},
```

Comment:
/home/travis/build/GoogleChrome/lighthouse/lighthouse-cli/test/smokehouse/smokehouse.js:174
      throw new Error(`Config did not trigger run of expected audit ${auditName}`);
      ^
Error: Config did not trigger run of expected audit time-to-interactive
    at auditNames.map.auditName (/home/travis/build/GoogleChrome/lighthouse/lighthouse-cli/test/smokehouse/smokehouse.js:174:13)